src/proxy-ui-api/frontend/tests/e2e/pages/ssMainPage.js

```diff
@@ -65,6 +65,21 @@ var navigateCommands = {
     this.click('@sessionExpiredPopupOkButton');
     return this;
   },
+  updateWSDLFileTo: function (newfile) {
+    const { exec } = require('child_process');
+    exec(
+      'ssh ' +
+        this.api.globals.testdata.split(':')[1].substring(2) + // remove protocol and port data
+        ' "cp ' +
+        this.api.globals.testfiles_path +
+        '/' +
+        newfile +
+        ' ' +
+        this.api.globals.testfiles_path +
+        '/testserviceX.wsdl"',
```

Comment:
We should use template literals instead of string concatenation. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals